sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOStorageReadTest.java

```diff
@@ -1351,4 +1435,613 @@ public void testReadFromBigQueryIOWithTrimmedSchema() throws Exception {
 
     p.run();
   }
+
+  @Test
+  public void testReadFromBigQueryIOArrow() throws Exception {
+    LOG.info("Entering arrow test");
```

Comment:
It looks like this was a debug log message that can be removed now?

Reply:
Done